src/index.js

```diff
@@ -1,15 +1,49 @@
+class Chronometer {
+  currentTime = 0;
+  intervalId = null;
+  start(printTimeCallback) {
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (printTimeCallback) printTimeCallback();
+    }, 1000);
+  }
+  getMinutes() {
+    return Math.floor(this.currentTime / 60);
+  }
+  getSeconds() {
+    return Math.floor(this.currentTime % 60);
+  }
+  computeTwoDigitNumber(value) {
+    let valueToModify = value.toString();
```

Comment:
Here, you didn't need to pass the `value` to a string, as you could just do string interpolation with it.
```
if (value < 10) return `0${value}`;
    else return `${value}`;
```